index.rst

```diff
@@ -25,8 +25,9 @@ Contents:
    schedulerdb.rst
    statusdb.rst
    releng101/index.rst
-   coding.rst
    jobs_not_starting.rst
+   best-practices/index
+   architecture/index
```

Comment:
nit: everything else has .rst extension, I vote for being consistent, but not a blocker.